lib/Mojolicious/Guides/Rendering.pod

```diff
@@ -332,6 +332,76 @@ ensure that they don't conflict with existing methods you may already have.
 A list of all built-in helpers can be found in L<Mojolicious::Plugin::DefaultHelpers> and
 L<Mojolicious::Plugin::TagHelpers>.
 
+=head2 Static files
+
+Static files are automatically served from the C<public> directories of the application, which can be customized with
+L<Mojolicious::Static/"paths">, or one of the C<DATA> sections from L<Mojolicious::Static/"classes">. And if that's not
+enough you can also serve them manually with L<Mojolicious::Plugin::DefaultHelpers/"reply-E<gt>static"> and
+L<Mojolicious::Plugin::DefaultHelpers/"reply-E<gt>file">.
+
+  use Mojolicious::Lite -signatures;
+
+  get '/' => sub ($c) {
+    $c->reply->static('index.html');
+  };
+
+  get '/some_download' => sub ($c) {
+    $c->res->headers->content_disposition('attachment; filename=bar.png;');
+    $c->reply->static('foo/bar.png');
+  };
+
+  get '/leak' => sub ($c) {
+    $c->reply->file('/etc/passwd');
+  };
+
+  app->start;
+
+=head2 Static assets
+
+While Mojolicious does not have any special support for frontend frameworks like L<Vue.js|https://vuejs.org> and
+L<React|https://reactjs.org>, the C<public/assets> directory is reserved for static assets created by bundlers like
```

Comment:
Is the directory "reserved"..? I think the wording makes it sound like you can't use this for anything else.

Reply:
It is "soft" reserved now. The app generator creates it and files in it have some special handling by the static file server.